emmet-core/emmet/core/vasp/calc_types/enums.py

```diff
@@ -1,18 +1,18 @@
 """
-Autogenerated Enums for VASP RunType, TaskType, and CalcType
-Do not edit this by hand. Edit generate.py or run_types.yaml instead
+Autogenerated Enums for VASP RunType, TaskType, and CalcType.
+
+Do not edit this by hand. Edit generate.py or run_types.yaml instead.
 """
 from emmet.core.utils import ValueEnum
 
 
 class RunType(ValueEnum):
-    """VASP calculation run types"""
+    """VASP calculation run types."""
 
     AM05 = "AM05"
     GGA = "GGA"
     PBE = "PBE"
     PBESol = "PBESol"
-    PBEsol = "PBEsol"
```

Comment:
@mjwen Why was this changed from PBEsol to PBESol? This (very minorly) breaks some of the validation code I'm working on. Also, it isn't consistent with the original PBEsol paper's capitalization (https://arxiv.org/abs/0711.0156)